grades/forms.py

```diff
@@ -1,6 +1,15 @@
 from django import forms
+from grades.models import Grade, Course
 
 
 class NewCourseForm(forms.Form):
     course_name = forms.CharField(max_length=100)
     credinitials = forms.IntegerField()
+
```

Comment:
In PR #67 user was added to the course model. (@or313 )

Reply:
Looks like the form will only be called by a logged in user. So this  comment isn't relevant.